examples/iam-access-groups.v2.test.js

```diff
@@ -81,79 +84,54 @@ describe('IamAccessGroupsV2', () => {
     // begin-create_access_group
 
     const params = {
-      accountId: testAccountId,
+      accountId: 'testString',
```

Comment:
These sorts of changes make the examples much less valuable to customers in terms of serving as an example that they can follow.
"testAccountId" was used to store a config property TEST_ACCOUNT_ID.
With this change, how were you able to run the examples cleanly???